cluster/addons/fluentd-elasticsearch/es-image/BUILD

```diff
@@ -22,7 +22,7 @@ go_library(
         "//vendor/k8s.io/client-go/rest:go_default_library",
         "//vendor/k8s.io/client-go/tools/clientcmd:go_default_library",
         "//vendor/k8s.io/client-go/tools/clientcmd/api:go_default_library",
-        "//vendor/k8s.io/klog:go_default_library",
```

Comment:
Flag for @liggitt. Want to make sure he's aware we're switching to klog/v2 as it has caused issues in the past.